.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,13 @@
 
 **Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
 
+**Testing done**:
+A PR must be marked "[WIP]", if no test result is provided. A WIP PR won't be reviewed, nor merged.
+The requester can determine a sufficient test, e.g. build for a cosmetic change, E2E test in a predeployed setup, etc.
+For new features, new tests should be done, in addition to regression tests.
```

Comment:
> For new features, new tests should be done, in addition to regression tests.

Do you mean the developer who is adding a new feature should add a new e2e test? It would be worth mentioning the requirement of adding a unit test rather.

Reply:
I would leave it up to the developer to decide how and when a test should be added, because it depends on the nature of the change.

You could add unit tests, FVT tests, integration tests, random tests, perf tests, etc. You could check in the tests before (TDD), with, or after the change. I do not hold strong opinions at the moment.